spec/fixtures/vcr/push_target_returns_count.yml

```diff
@@ -0,0 +1,46 @@
+---
+http_interactions:
+- request:
+    method: get
+    uri: https://example.com/rest_api/v1/push%2Dtargets/
+    body:
+      encoding: US-ASCII
+      string: ''
+    headers:
+      Accept:
+      - application/json
+      Content-Type:
+      - application/json
+      User-Agent:
+      - Faraday v0.9.2
+  response:
+    status:
+      code: 200
+      message: 
+    headers:
+      date:
+      - Mon, 29 Jan 2018 03:34:55 GMT
+      server:
+      - WSGIServer/0.1 Python/2.7.14
+      expires:
+      - Mon, 29 Jan 2018 03:35:25 GMT
+      vary:
+      - Accept, Cookie
+      last-modified:
+      - Mon, 29 Jan 2018 03:34:55 GMT
+      allow:
+      - GET, POST, PUT, PATCH, DELETE, HEAD, OPTIONS
+      cache-control:
+      - max-age=30
+      x-frame-options:
+      - SAMEORIGIN
+      content-type:
+      - application/json
+    body:
+      encoding: UTF-8
+      string: '{"count":2,"next":null,"previous":null,"results":[{"id":1,"name":"cdn-qa","description":"CDN
```

Comment:
Patch looks good to me, but I think we'd better not put the 'https://rhn-qa.redhat.com' in our test data, as this is open source package, you can let it be null or other test hostname.

Reply:
Agreed. I'll make the change. Thank-you for the prompt review.

Reply:
Cool and thanks for commit this nice patch, I forgot to add this endpoint when I added the attribute allowed_push_targets in release and release-variant. :)